### DISCUSSION TITLE
Daylight savings time

### DISCUSSION THREAD

**Discussion by holgergp** [starship/starship#4173] [Q&A]
Hi all,

maybe a stupid question. But hopefully you can point me in the right direction.
Is it possible to tell Starship about Daylight Savings Time? For now I manually edit the `utc_time_offset` when this occurs.
How do you do it? What am I missing?

Thanks for your support and for a great tool!

Cheers!

Holger

**Comment by davidkna** [comment]
The default value of `local` should respect DST. If that doesn't work for you, please create a feature request for an option to set a timezone name like `Europe/Berlin`, which AFAIK should include DST information.

**Comment by NewtonChutney** [comment]
Further, one can tweak the timezone by putting this in their shell config:
`export TZ="Europe/Berlin"`

Especially useful if they don't have root perms to change the machine's tz
